src/Blazor.Gitter.Core/Components/Shared/RoomMessages.razor.cs

```diff
@@ -28,7 +28,7 @@ public class RoomMessagesBase : ComponentBase, IDisposable
         bool IsFetchingOlder = false;
         bool NoMoreOldMessages = false;
         bool FirstLoad = true;
-        internal bool Paused = true;
+        internal bool Paused = false;
```

Comment:
I've set this to false as otherwise a pause overlay is shown when a room is first loading which isn't true. I couldn't see any issue with this but I wanted to point it out.